src/js/base/runtime.js

```diff
@@ -285,7 +285,7 @@ function (Namespace, jsnums, codePoint, util, exnStackParser, loader, seedrandom
         }
         s += " }";
         return s;
-        
+
```

Comment:
Realistically, all this whitespace tweaking shouldn't be part of this PR, since it just increases the likelihood of merge conflicts when we try to cherry-pick it